MCDataProducts/inc/CaloShowerStep.hh

```diff
@@ -22,56 +20,47 @@
 namespace mu2e {
 
 
-   class CaloShowerStep {
-
-
-       public:
-	 
-	  CaloShowerStep(): volumeId_(-1),sim_(),nCompress_(0),time_(0),energyMC_(0),momentumIn_(0),posIn_(),pos_(),covPos_() {}
+   class CaloShowerStep 
+   {
+       public:	 
+	  CaloShowerStep(): volumeId_(-1),sim_(),nCompress_(0),time_(0),energyDepG4_(0),energyDepBirks_(0),momentumIn_(0),pos_() {}
 	  
-	  CaloShowerStep(int volumeId, art::Ptr<SimParticle> const& sim, int nCompress, double time,  
-	                 double energy, double momentumIn, const CLHEP::Hep3Vector& posIn, const CLHEP::Hep3Vector& pos, 
-			 const CLHEP::HepSymMatrix& covPos): 
+	  CaloShowerStep(int volumeId, const art::Ptr<SimParticle>& sim, int nCompress, float time,  
+	                 float energyDepG4, float energyDepBirks, float momentumIn, const CLHEP::Hep3Vector& pos): 
 	    volumeId_(volumeId),
 	    sim_(sim),
 	    nCompress_(nCompress),
 	    time_(time),
-	    energyMC_(energy),
+	    energyDepG4_(energyDepG4),
+	    energyDepBirks_(energyDepBirks),
 	    momentumIn_(momentumIn),
-	    posIn_(posIn), 
-	    pos_(pos), 
-	    covPos_(covPos)	  
-	    {}
-
-
-	  int                            volumeId()    const {return volumeId_;}
```

Comment:
VolumeID refers to the Geant4 volume for the crystals, which is a fixed ID (set in the disk calorimeter maker function). It is only used in CaloShowerStep and CaloShowerROMaker (and a few print functions that dump the object content). 
Do we really need to make it a class and complicate this? Other classes uses volumeID from StepPointMCs as an int. Unless this is really needed, I would refrain of doing it.

Reply:
In MDC2018 there was confusion between energy deposits in crystals vs sipms, that caused problems multiple times.  If you say you've fixed that some other way this isn't mandatory.   If the Id appears in a public interface (for instance to access calibrations) a class would add value.

Reply:
As discussed, I have renamed the accessor "SiPMID" and "crystalID". They are not their own class since 1) we used them to reference indices in vectors for fast algorithms quit often, so a dedicated class would need a method asInt() that would be written everywhere, and 2) there shouldn't be any source of confusion with these new names.